Makefile

```diff
@@ -1,6 +1,23 @@
 #!/usr/bin/make -f
 
-TARGETARCH=$(shell gcc -dumpmachine)
+CC         ?= gcc
+INSTALL    ?= install
+PKG_CONFIG ?= pkg-config
+RONN       ?= ronn
+
+CFLAGS     ?= -O2 -g
+
+prefix         ?= /usr/local
+sbindir        ?= $(prefix)/sbin
+datadir        ?= $(prefix)/share
+mandir         ?= $(datadir)/man
```

Comment:
Yes they belong in $(mandir)/man8/, but I don't think he should change the mandir variable, instead they should install to  $(mandir)/man8/ instead of changing the mandir variable.

Reply:
That makes sense to me, I'll amend my suggestion.

Reply:
Nah... the expected value for mandir is `$(datadir)/man`. This should rather be better handled in install rules. My mistake that I've overlooked it initiall. Thanks for heads-up.

fixed